addons/web/static/src/legacy/js/core/utils.js

```diff
@@ -1140,9 +1140,11 @@ var utils = Object.assign({
             ['res_model', '=', 'ir.ui.view'],
             '|',
             '|',
-            ['name', '=like', '%.assets\_%.css'],
-            ['name', '=like', '%.assets\_%.js'],
+            '|',
+            ['name', '=like', '%.%assets\_%.css'],
```

Comment:
why do you need to change `'%.assets\_%.css'` to `'%.%assets\_%.css'` ?
I see no assets on your screenshot for which the second `%` is necessary

Reply:
You can second attachment

![Screenshot from 2022-03-10 11-31-52](https://user-images.githubusercontent.com/72728567/157599489-c8167930-eb29-48de-9205-e50328ac9c56.png)

Reply:
ok, thanks for the explanation :)